docs/src/pages/components/data-grid/columns/RenderExpandCellGrid.tsx

```diff
@@ -121,7 +121,7 @@ function renderCellExpand(params: GridCellParams) {
   return (
     <GridCellExpand
       value={params.value ? params.value.toString() : ''}
-      width={params.colDef.width}
+      width={params.colDef.computedWidth}
```

Comment:
Should this variable be in the public API landscape? Yeah, maybe.

Reply:
I think so
Otherwise, custom cell renderer that really need the width of the cell would need to capture it with observers of other complex approach which seems counterproductive.